org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/wizard/project/GradleProjectWizardPage.java

```diff
@@ -99,6 +99,19 @@ private void bindToConfiguration() {
             public void modifyText(ModifyEvent e) {
                 File projectDir = FileUtils.getAbsoluteFile(GradleProjectWizardPage.this.projectDirText.getText()).orNull();
                 getConfiguration().setProjectDir(projectDir);
+                // TODO currently a
```

Comment:
Talked with @donat that the aim is different here, because I'd just want to show a warning rather than an error, which is currently not supported by a Validator. Eclipse usually works with an org.eclipse.core.runtime.IStatus to distinguish between different status, like errors and warnings. Therefore we could add a derived buildship specific validator, which can also works IStatus objects or find a different approach for the tooling api to distinguish between certain states.
@etiennestuder What do you think about it?